src/pipeline.ts

```diff
@@ -198,7 +203,9 @@ export async function searchForLocalTorrentByCriteria(
 	const hashesToExclude = await getInfoHashesToExclude();
 	let matches = 0;
 	for (const searchee of searchees) {
-		if (!filterByContent(searchee)) return null;
+		if (!handleFilteringSonarrLibrary(searchee)) continue;
+		else if (!filterByContent(searchee)) return null;
```

Comment:
To address this and the above, if we move this into filterByContent it will return null, which we don't want - if handleFilteringSonarrLibrary continues the search still "succeeds" - only omitting what would otherwise fail in filterByContent